crates/biome_configuration/src/diagnostics.rs

```diff
@@ -7,9 +7,42 @@ use biome_rowan::SyntaxError;
 use serde::{Deserialize, Serialize};
 use std::fmt::{Debug, Formatter};
 
-/// Series of errors that can be thrown while computing the configuration
+/// Series of errors that can be thrown while computing the configuration.
 #[derive(Deserialize, Diagnostic, Serialize)]
 pub enum ConfigurationDiagnostic {
+    /// Diagnostics related to `biome.json` files
+    Biome(BiomeDiagnostic),
+    /// Diagnostics related to `.editorconfig` files
+    EditorConfig(EditorConfigDiagnostic),
+}
+
+impl Debug for ConfigurationDiagnostic {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        std::fmt::Display::fmt(self, f)
+    }
+}
+
+impl std::fmt::Display for ConfigurationDiagnostic {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        self.description(f)
+    }
+}
```

Comment:
A couple of things:
- Usually, you implement the `Display` and use `std::fmt::Debug`. Never the other way around, because `Debug` is meant to print info in debug mode, so we want to display everything. 
- I don't think we need `Display` for diagnostics. Is there a specific reason why you added it?

Reply:
I was just replicating what was already there, I have no idea why it was like that.

Reply:
Ooooh I see now. `Display` is needed for `impl Error` :/